packages/compass-import-export/src/modules/import.ts

```diff
@@ -240,11 +229,11 @@ export const startImport = (): ImportThunkAction<Promise<void>, AnyAction> => {
     let errorLogFilePath;
     let errorLogWriteStream: fs.WriteStream | undefined;
     try {
-      errorLogFilePath = await getErrorLogPath(fileName);
-
-      errorLogWriteStream = errorLogFilePath
-        ? fs.createWriteStream(errorLogFilePath)
-        : undefined;
+      const fs = new Filesystem({
+        subdir: 'ImportErrorLogs',
+      });
+      const errorLogFileName = `import-${path.basename(fileName)}.log`;
+      errorLogWriteStream = await fs.createWriteStream(errorLogFileName);
```

Comment:
Maybe not a topic for this PR, but I do wonder if this should be rather written to some tmp / log folder instead of user data folder and as such whether createWriteStream is even an api we need for this fs utility we are adding

Reply:
This is a good point. I'll undo changes to this package and keep it as is and not expose `createWriteStream` from the api. I'll add it to the technical alignment sync to discuss within the team.